tests/OpenCFP/Domain/Services/ResetEmailerTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace OpenCFP\Tests\Domain\Services;
+
+use OpenCFP\Domain\Services\ResetEmailer;
+
+class EmailerTest extends \PHPUnit_Framework_TestCase
+{
+    private $mailer;
+    private $template;
+    private $config_email;
+    private $config_title;
+
+    private $user_id;
+    private $user_email;
+    private $reset_code;
+
+    public function setUp()
+    {
+        $this->mailer = \Mockery::mock('mailer')->shouldReceive('send')->once()
+            ->with(\Mockery::on($this->validateEmail()))->getMock();
+
+        $this->template = \Mockery::mock('template')->shouldIgnoreMissing();
+        $this->config_email = 'admin@example.com';
+        $this->config_title = 'Reset';
+
+        $this->user_email = 'user@example.com';
+        $this->user_id = 123;
+        $this->reset_code = '987abc';
```

Comment:
I put `user_email` up here so that we can easily grab it in `validateEmail()`; we don't really need the other two, unless we decide to add those assertions against the created Swift message